Revit_Engine/Query/InnerPolyCurves.cs

```diff
@@ -38,10 +34,10 @@ public static partial class Query
         /****              Public methods               ****/
         /***************************************************/
 
-        [Description("Returns the inner PolyCurves from list. QUICK CHECK (BOUNDING BOX)")]
-        [Input("polyCurve", "PolyCurve")]
-        [Input("polyCurves", "PolyCurves")]
-        [Output("PolyCurves")]
+        [Description("Returns the PolyCurves from list that lie inside given outer PolyCurve. QUICK CHECK (BOUNDING BOX).")]
```

Comment:
```suggestion
        [Description("Returns the PolyCurves from the list 'polyCurves' that lie inside the provided PolyCurve. QUICK CHECK (BOUNDING BOX).")]
```

Reply:
I have updated the description, but would be worried about that tbh... This method is legacy, first one to be deprecated in the next Milestone (1. it should sit in geometry 2. it is a duplicate of geometry 3. it is case-specific).